### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by myxxmikeyxx** [pdq21/WinEP-misc#1]
For using mdm-remotewipe, are you able to pass an unattended.xml file in so it will auto configure windows and the setup?

**Comment by pdq21** [comment]
Hello. Do you mean to transfer the xml to the EP before wiping and automatically starting the unattended setup?

**Comment by myxxmikeyxx** [comment]
Yes. I assume it might not be a thing or work. I have a very special use case. Does that script allow me to remote wipe any managed windows devices?
Long Story:
I work at a place that has remote management software installed on all our devices. I want to be able to remotely send a wipe command and give it an unattended XML file for the setup process. I would be doing them all in the building so it's not a super big deal just would be nice to plug the machine in and go and not have to touch all of them. Right now we have to plug every device into ethernet, boot to nic, and use MDT for imaging with a task for unattended and post software install.
